core/state_transition.go

```diff
@@ -108,6 +111,24 @@ func IntrinsicGas(data []byte, accessList types.AccessList, isContractCreation b
 			gas += lenWords * params.InitCodeWordGas
 		}
 	}
+
+	// This gas is used for charging user for one `debitFrom` transaction to deduct their balance in
```

Comment:
We can "reshape" this comment with the new way of debit/credit and not the legacy one